meshroom/ui/qml/Viewer/FeaturesInfoOverlay.qml

```diff
@@ -16,16 +16,12 @@ FloatingPane {
     property int pluginStatus: Loader.Null
     property Item featuresViewer: null
     property var mfeatures: null
-    property var featureExtractionNode: null
+    property var mtracks: null
+    property var msfmdata: null
 
     ColumnLayout {
         // Header
         RowLayout {
-            // FeatureExtraction node name
-            Label {
-                text: featureExtractionNode ? featureExtractionNode.label : ""
-                Layout.fillWidth: true
-            }
```

Comment:
As the name of the active `FeatureExtraction` node is not displayed anymore in the FeatureViewer, it is impossible to know what the viewer is showing when there are several branches / `FeatureExtraction` nodes. I understand that using the `FeatureExtraction` node alone is not good solution _per se_ as the Feature Viewer does not only display information coming from the `FeatureExtraction` node, but we need something to indicate what we are displaying.